src/tests/libecalc/integration/snapshots/test_all_energy_usage_models/test_all_results/all_energy_usage_models_v3.json

```diff
@@ -22145,48 +22145,34 @@
                 {
                     "energy_usage": {
                         "timesteps": [
-                            "2018-01-01 00:00:00",
-                            "2019-01-01 00:00:00",
-                            "2020-01-01 00:00:00"
+                            "2018-01-01 00:00:00"
```

Comment:
Probably easiest to see the fix here. This is the first model result which was previously mutated to the aggregated result, now it's only defined for 2018, and the next model result continues in 2019.

Reply:
Yes, now i understand :)